spec/data-model/README.md

```diff
@@ -81,7 +81,7 @@ The `value` of `Text` is the "cooked" value (i.e. escape sequences are processed
 Implementations MUST NOT rely on the set of `Expression` `body` values being exhaustive,
 as future versions of this specification MAY define additional expressions.
 If encountering a `body` with an unrecognised value,
-an implementation SHOULD treat it as it would a `Reserved` value.
+an implementation SHOULD treat it as it would an `Unsupported` value.
```

Comment:
Consider using a more positive statement:

```suggestion
A `body` with an unrecognized value SHOULD be treated as
an `Unsupported` value.
```

Will `body` always have a sigil, that is, will it fit the description below in which unsupported is always reserved or private use? Or do we need to add text below that says (effectively) "reserved, unsupported private-use, or any unrecognized sequence"